app/src/main/java/ar/com/wolox/android/example/network/ServerComunication.java

```diff
@@ -0,0 +1,24 @@
+package ar.com.wolox.android.example.network;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+import java.util.List;
+
+/**
+ * Server Comunication Class
+ */
+
+public class ServerComunication {
+    @SerializedName("users")
+    @Expose
```

Comment:
I think we could remove all this `@Expose`, or I'm wrong? (I see this in other places, take care of them too)